### ISSUE TITLE
[$10][MSFT-208] The "Code/ TCO19" icons doesn't have highlight when focus by keyboard

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2871]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges/30095347?tab=details
2. Press tab key till the "Code/ TCO19" icons is selected

**Expected Result**
The "Code/ TCO19" icons should have blue highlight when focus by keyboard

**Actual Result**
The "Code/ TCO19" icons doesn't have highlight when focus by keyboard

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: 
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F18

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095969 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095969 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095969 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu it requires change on another repo, topcoder-react-ui-kit.

PR for same https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/7

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2871 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2871 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2871 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097114 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097114 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu @codeMinter PR branch changed.topcoder-platform/topcoder-react-ui-kit#7

**Comment by PrakashDurlabhji** [comment]
@veshu just sometime updating

**Comment by PrakashDurlabhji** [comment]
@veshu @codeMinter new PR https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/20

**Comment by PrakashDurlabhji** [comment]
@veshu can you please double the prize it was left out while you doubled others. thank you

**Comment by drasticdpk** [comment]
Verified on 
Operating System: Windows
Browser: Chrome

**Comment by drasticdpk** [comment]
Here is the reference :
https://monosnap.com/file/Lkll9WXQpvpVjpxUBYHBrDOgVpaTae

**Comment by veshu** [comment]
@PrakashDurlabhji Please update the events tags https://github.com/topcoder-platform/topcoder-react-ui-kit/tree/qa-accessibility/src/shared/components/tags/event

e.g. for development `color` should be changed from `$tc_green` to `$tc_green_110` https://prnt.sc/one12j 
similar for design and data-since replace the color with 110 variant . 

I will raise the prize, but please complete within next few hours Thanks!

**Comment by PrakashDurlabhji** [comment]
@veshu sure making a PR now

**Comment by PrakashDurlabhji** [comment]
@veshu new  PR again https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/24

**Comment by veshu** [comment]
@PrakashDurlabhji The PR is wrong, you have changed the hover color instead of color at normal state. See the image I have highlighted

>e.g. for development color should be changed from $tc_green to $tc_green_110 https://prnt.sc/one12j

Also for blue please use `tc_dark_blue_110`.

**Comment by PrakashDurlabhji** [comment]
@veshu PR updated topcoder-platform/topcoder-react-ui-kit#24

**Comment by tosha5252** [comment]
'TCO19' button was not highlighted blue.

tested at: 
https://www.topcoder.com/challenges/30098257

Windows 10
Browser: Chrome

**Comment by veshu** [comment]
@tosha5252 looks like you are testing on prod where as this is only been deployed to dev.

@sushilshinde Can we move this issue to prod as there won't be tco tag on dev/test.

**Comment by tosha5252** [comment]
@veshu looks like it had already been moved to prod and then back to dev...I retested at the link that has been tested before...since there are no sample pages in the dev env with the 'TCO19' icon on it...can one be added so that it can be tested there?

**Comment by veshu** [comment]
@tosha5252 The track tag was on prod but not 'TCO19'.

We tried to add tco in dev but it wasn't successful. So we need to test that on prod itself. 

Please wait for sushil move this issue to production. Thanks!

**Comment by sushilshinde** [comment]
Moving this to prod

**Comment by tosha5252** [comment]
Tested in prod...https://www.topcoder.com/challenges/30098257

TCO19 tag not highlighted

Windows  env
Browser: Chrome

**Comment by tosha5252** [comment]
Tested in prod...https://www.topcoder.com/challenges/30098257
In Chrome: TCO19 tag not highlighted at all
In Edge: There is a slight color change to the wording 'TCO19' (see video)

[tco19.zip](https://github.com/topcoder-platform/community-app/files/3480342/tco19.zip)



Windows env
Browser: Chrome

**Comment by PrakashDurlabhji** [comment]
@veshu this was reason I added color change to ".hover" selector in css.
but then due to your comment I reverted my code to match as suggested by you.
shall I add that logic back and make a PR?

reason is because there is no visible difference in color between those green shades hence may be 
"TCO 19" focus is not noticeable

PR which was earlier raised: https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/24/commits

@tosha5252

**Comment by veshu** [comment]
@PrakashDurlabhji Can you please update .hover background-color to their 30 level variant.
for development it will be $tc_green_30 instead of $tc_green_10 ?

**Comment by PrakashDurlabhji** [comment]
@veshu @tosha5252  new Pr according to above comment https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/25

**Comment by sushilshinde** [comment]
Made a release on npm

**Comment by tosha5252** [comment]
Verified changed in prod

Win10/Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097114<br/><br/>```This is an automated message for crazyk via Topcoder X```
